### DISCUSSION TITLE
Please make the docs easier to use for a specific version

### DISCUSSION THREAD

**Discussion by darylknight** [solspace/craft-calendar#179] [Feature Requests]
**Describe the bug or issue you're experiencing**
Sorry there was no option for a discussion topic here, so filing an issue. I'm working on a site right now where I needed to lookup `loadOccurrences`. When I type that into the solspace calendar docs searchbar, I get this:

![image](https://user-images.githubusercontent.com/1250017/200518816-ececf502-e71d-47db-9b79-9a7a59fcf37d.png)

If I hover over those links, which are all identical, my browser tells me that they're for Calendar v1, v2 and v4. Would you consider updating the docs here so that we can search for only relevant docs on the same version? This applies to all Solspace products, and even shows doc pages from ExpressionEngine soometimes, which is not that useful.

**Comment by kjmartens** [comment]
Thanks for your feedback, @darylknight. This is something we're aware of and will see if there's something we can do to improve it.
